cmd/horcrux/cmd/signer.go

```diff
@@ -60,6 +69,8 @@ func StartSignerCmd() *cobra.Command {
 				return err
 			}
 
+			fmt.Println(singleSignerWarning)
```

Comment:
This should be using `fmt.Fprintln(cmd.OutOrStdout(), ...)` for more straightforward testing of cobra commands later.